products/opensearch.md

```diff
@@ -24,13 +24,26 @@ releases:
 
 ---
 
-> [OpenSearch](https://opensearch.org/) is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. Developers build with OpenSearch for use cases such as application search, log analytics, data observability, data ingestion, and more.
+> [OpenSearch](https://opensearch.org/) is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest,
+> search, visualize, and analyze data. Developers build with OpenSearch for use cases such as application search, log analytics,
+> data observability, data ingestion, and more.
+
+## Official Release schedule
+
+For more information on the changes planned for each release, see:
```

Comment:
A summary will be helpful here, instead of just links.

Reply:
@adriens ping :)

Reply:
I'll work on release schedule :+1: